src/Engine/ProtoCore/Lang/FunctionEndPoint.cs

```diff
@@ -61,6 +61,9 @@ public System.Type CLRReturnType
             }
         }
 
+        
+        internal Attribute[][] ParamAttributes => method.GetParameters().Select(x => x.Info.GetCustomAttributes().ToArray()).ToArray();
```

Comment:
Where is this used?

Reply:
it's not currently - we could get rid of it as you suggested, I instead used the `procedureNode.ArgumentTypes`... I can send a cleanup PR for this.